R/module_teal_navigation.R

```diff
@@ -491,3 +462,123 @@ srv_teal_module.teal_module <- function(id,
     ...
   )
 }
+
+#' @keywords internal
+.get_breadcrum <- function(module) {
```

Comment:
It is not a "getter" but it makes stuff.
```suggestion
.modules_breadcrumb <- function(module) {
```